### ISSUE TITLE
@nrwl/next:build disregards project.json 'outputPath' and --outputPath CLI option

### ISSUE THREAD

**Issue by kentytik** [nrwl/nx#15471]
### Current Behavior

@nrwl/next:build disregards project.json 'outputPath' and --outputPath CLI option and outputs the .next directory generated from the build into into the root directory, rather.

### Expected Behavior

@nrwl/next:build must output the .next directory generated from the build into the directory setting for either (i) 'outputPath' in project.json or (ii) --outputPath CLI option

### Steps to Reproduce

1. Nextjs app generated with the NX console
2. Build production app with 'npx nx build <APP NAME>'
3. 
3a. the .next directory is generated in the directory for 'options > root'  rather than in the directory for either 'options > outputPath' or (b) '--outputPath' CLI option
3b. 'npx nx export <APP NAME>' fails with the message "_Could not find a production build in the .next' directory that should be in the 'dist' directory . Try building your app with 'next build' before starting the static export. https://nextjs.org/docs/messages/next-export-no-build-id_"

### Nx Report

```shell
Node : 18.14.2
   OS   : linux x64
   npm  : 9.6.0

   nx                      : 15.8.5
   @nrwl/js                : 15.8.5
   @nrwl/jest              : 15.8.5
   @nrwl/linter            : 15.8.5
   @nrwl/workspace         : 15.8.5
   @nrwl/angular           : 15.8.5
   @nrwl/cli               : 15.8.5
   @nrwl/cypress           : 15.8.5
   @nrwl/devkit            : 15.8.5
   @nrwl/eslint-plugin-nx  : 15.8.5
   @nrwl/express           : 15.8.5
   @nrwl/nest              : 15.8.5
   @nrwl/next              : 15.8.5
   @nrwl/node              : 15.8.5
   @nrwl/react             : 15.8.5
   @nrwl/rollup            : 15.8.5
   @nrwl/tao               : 15.8.5
   @nrwl/web               : 15.8.5
   @nrwl/webpack           : 15.8.5
   typescript              : 4.9.5
   ---------------------------------------
   Community plugins:
   @ngrx/effects               : 15.3.0
   @ngrx/router-store          : 15.3.0
   @ngrx/store                 : 15.3.0
   @nguniversal/express-engine : 15.2.0
   @ngrx/store-devtools        : 15.3.0
   @nguniversal/builders       : 15.2.0
```

### Failure Logs

```shell
NX   Could not find a production build in the '.../.next' directory. Try building your app with 'next build' before starting the static export. https://nextjs.org/docs/messages/next-export-no-build-id
```

**Comment by kentytik** [comment]
My workaround now is to create a symlink (on Linux) from the '.next' directory in the project root to the '.next' directory in the dist directory. That's working so far with all nx targets working.

**Comment by ianldgs** [comment]
Probably a duplicate of #15214

**Comment by kentytik** [comment]
> Probably a duplicate of #15214

It is, thanks.
